app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -357,9 +357,6 @@ public void onDestroy() {
         if (Pokefly.isRunning()) {
             stopService(new Intent(MainActivity.this, Pokefly.class));
         }
-        if (screen != null) {
-            screen.exit();
-        }
```

Comment:
When I first read this code, I tought this was an incorrect behavior.
I think Pokefly should run even if MainActivity gets destroyed.
@sarav I don't remember about your request, maybe I just lost it between the "notification" PR comments...
Anyway, +1 for me!